README.md

```diff
@@ -1,6 +1,20 @@
 # sanitizer
 Identify and sanitize sensitive information.
+
+## Support
+The following browsers are supported:
+ - Google Chrome
+ - Firefox
+
 At the moment, it sanitizes HAR files.
 Support for sanitizing SAML requests/responses are in progress.
 
+## Build
+To build the WASM script, run the `build_was` script.
```

Comment:
I'm getting an error when only running this command:
```
# sanitizer
./sanitize.go:55:27: undefined: jsonpath.GetWithPaths
```
Are there any prerequisites to build the WASM script?

Reply:
A version of `jsonpath` with `GetWithPaths` hasn't been released yet.

Use the `master` branch tip for installing `github.com/PaesslerAG/jsonpath`.
Something like:
```
go get github.com/PaesslerAG/jsonpath@master
```

The generated `index.html` and WASM code are stand alone and should work for you irrespective of the presence of the right dependency in your Go development environment.